### PR TITLE
test: Rule 45 Playwright Tests

### PR DESCRIPTION
**Pull request by Joseph2910**
## Description
End to End tests for this https://nhsd-jira.digital.nhs.uk/browse/DTOSS-9873

## Type of changes

- [X] Refactoring (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [X] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [X] I have followed the code style of the project
- [X] I have added tests to cover my changes
- [X] I have updated the documentation accordingly
- [X] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [X] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

**Review by SamRobinson75684 — DISMISSED** [review]

**Review by adamsheldon1 — DISMISSED** [review]

File: application/CohortManager/src/Functions/ScreeningValidationService/LookupValidation/Breast_Screening_lookupRules.json [hunk 1]
```diff
@@ -40,23 +40,15 @@
         "RuleName": "45.GPPracticeCodeDoesNotExist.BSSelect.NonFatal",
         "LocalParams": [
           {
-            "Name": "EnglishPostingCategory",
-            "Expression": "dbLookup.RetrievePostingCategory(newParticipant.CurrentPosting) == \"ENGLAND\""
+            "Name": "AcceptedCurrentPosting",
+            "Expression": "dbLookup.RetrievePostingCategory(newParticipant.CurrentPosting) != null && ((dbLookup.RetrievePostingCategory(newParticipant.CurrentPosting) == \"ENGLAND\" || dbLookup.RetrievePostingCategory(newParticipant.CurrentPosting) == \"ENG\" || dbLookup.RetrievePostingCategory(newParticipant.CurrentPosting) == \"IM\" || dbLookup.RetrievePostingCategory(newParticipant.CurrentPosting) == \"DMS\"))"
```
Comment (SamRobinson75684) [inline application/CohortManager/src/Functions/ScreeningValidationService/LookupValidation/Breast_Screening_lookupRules.json:44]: this expression while it might do what we want it to do could please split it up so that when we come to maintain it in the future it's much easier to understand what it is trying to do. See other rules for examples of when we combine multiple logical expressions or reach out if you need any nudges

**Review by adamsheldon1 — DISMISSED** [review]

File: application/CohortManager/src/Functions/ScreeningValidationService/LookupValidation/Breast_Screening_lookupRules.json [hunk 2]
```diff
@@ -33,18 +33,14 @@
...
-            "Name": "PrimaryCareExists",
-            "Expression": "dbLookup.CheckIfPrimaryCareProviderExists(newParticipant.PrimaryCareProvider)"
-          }, 
           {
             "Name": "IsExcluded",
             "Expression": "dbLookup.CheckIfPrimaryCareProviderInExcludedSmuList(newParticipant.PrimaryCareProvider)"
           }
         ],
-        "Expression": "(InvalidPostingCategory || string.IsNullOrEmpty(newParticipant.PrimaryCareProvider) || PrimaryCareExists || IsExcluded)",
+        "Expression": "(InvalidPostingCategory || ValidPrimaryCareProvider || IsExcluded)",
```
Comment (stephhou) [inline application/CohortManager/src/Functions/ScreeningValidationService/LookupValidation/Breast_Screening_lookupRules.json:43]: For this rule we need to make sure that the Primary care provider is not null. does ValidPrimaryCareProvider do that? and do we need a test for primary care provider is null?
